nibabel/freesurfer/io.py

```diff
@@ -160,6 +160,29 @@ def read_morph_data(filepath):
     return curv
 
 
+def write_morph_data(filepath, values):
+    """Write out a Freesurfer morphometry data file.
+
+    See:
+    http://www.grahamwideman.com/gw/brain/fs/surfacefileformats.htm#CurvNew
+
+    Parameters
+    ----------
+    filepath : str
```

Comment:
Yup, then something in the description to say parameter can be filename string or open file-like object with constraints, which in your case are that the object has to be open in binary write mode and implement the `write` method (I suppose, for the array `tofile` method).